package.json

```diff
@@ -5,7 +5,7 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/HubSpot/hubspot-cli",
   "dependencies": {
-    "@hubspot/local-dev-lib": "3.4.1",
+    "@hubspot/local-dev-lib": "0.3.0-experimental.0",
```

Comment:
Reminder to update once https://github.com/HubSpot/hubspot-local-dev-lib/pull/255 is merged and published